packages/chip/README.md

```diff
@@ -0,0 +1,59 @@
+# Chip
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/chip.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/chip/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/chip
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/chip
+```
+
+## Example
+
+```js
+import { Chip } from `@leafygreen-ui/chip`;
+
+<Chip label="leafygreen" />
+
+or
+
+<Chip
+  label="leafygreen"
+  variant="blue"
+  size="large"
+  disabled
+  onDismiss={() => {}}
+  popoverZIndex={1}
+  chipCharacterLimit={10}
+  chipTruncationLocation="end"
+  dismissButtonAriaLabel="aria-label"
+  darkMode
+/>
+
+```
+
+## Properties
+
+| Prop                     | Type                                                                                 | Description                                                                                                                                                                                                                                               | Default             |
+| ------------------------ | ------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- |
+| `label`                  | `React.ReactNode`                                                                    | Label rendered in the chip                                                                                                                                                                                                                                |                     |
+| `chipTruncationLocation` | `'end'` \| `'middle'` \| `'none'` \| `'start'`                                       | Defines where the ellipses will appear in a Chip when the label length exceeds the `chipCharacterLimit`. If `none` is passed, the chip will not truncate. **Note**: If there is any truncation, the full label text will appear inside a tooltip on hover | `none`              |
```

Comment:
I don't know if none should be an option. i think we should set a default and if chipCharacterLimit is true, then we truncate. I don't like a boolean being based on two props, when one should be enough.